core/src/main/kotlin/org/evomaster/core/search/service/Archive.kt

```diff
@@ -385,7 +385,13 @@ class Archive<T> where T : Individual {
             }
 
             val limit = apc.getArchiveTargetLimit()
-            if (current.size < limit) {
+
+            /**
+             individual can be added only if the target k is not covered.
```

Comment:
inside a function, no need to use JavaDoc notation `/**`, just use `/*`

Reply:
modified